batch-processing-massive-data-much-quicker-with-tispark.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Batch Processing Massive Data Much Quicker with TiSpark
+author: ['Zhexuan Yang']
+date: 2020-12-22
+summary: A senior solution architect at PingCAP explains why TiSpark (TiDB + Apache Spark) is better than the traditional batch processing solution, how you can benefit from TiSpark, and how it works.
+tags: ['TiSpark']
+categories: ['Engineering']
+image: /images/blog/batch-processing-massive-data-much-quicker-with-tispark.jpg
+aliases: ['/blog/batch-processing-massive-data-much-quicker-with-tispark']
+---
+
+![Batch Processing Massive Data Much Quicker with TiSpark](media/batch-processing-massive-data-much-quicker-with-tispark.jpg)
+
+TiSpark is an Apache Spark plugin that works with the TiDB platform to answer complex online analytical processing (OLAP) queries. It is widely used by our customers to batch process their massive data and discover analytical insights.
```

Comment:
Please add a link for "TiSpark" and "TiDB."
I also suggest linking "customers" to https://en.pingcap.com/case-studies.

Reply:
done